### PR TITLE
v0.10.1

### PR DESCRIPTION
**Pull request by SC426**
[Upstream](https://github.com/xattr/xattr)
[Changelog](https://github.com/xattr/xattr/blob/master/CHANGES.txt)

### Actions
- Update version number and sha256
- Linter fixes
- Recipe cleanup

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
Just curious to know why we are skipping the build for `python 3.7`.

**Review by skupr-anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,42 +14,42 @@ source:
 
 build:
   skip: true  # [win]
+  skip: true  # [py<38]
```
Comment (danpetry) [inline recipe/meta.yaml]: `fn: {{ name }}-{{ version }}.{{ bundle }}` doesn't do anything, you can delete

Comment (danpetry) [inline recipe/meta.yaml]: please clarify why `[py<38]`, I can't find anything upstream. Note that skipping certain python versions should be specified only if there's some requirement upstream. It shouldn't reflect _anaconda's_ policy about which python versions we build for.

Comment (SC426) [reply recipe/meta.yaml]: It was a mistake on my part as I assumed there needed to be some kind of skip. I will delete it.

Comment (SC426) [reply recipe/meta.yaml]: Fixed, thanks!

**Review by danpetry — APPROVED** [review]

**Pull request merged**